src/docs/firmware/write-protect.md

```diff
@@ -11,7 +11,7 @@ You do not have to disable write protect if flashing [RW_Legacy](about.md#rw_leg
 Refer to the [supported devices list](supported-devices.md) to figure out which method you need to use to disable write-protect on your device.
 
 Depending on your device, you will need to do **one** of the following:
-- Take out a WP screw
-- [Unplug your battery](battery.md)
-- Bridge two jumpers
-- [Use a SuzyQable](suzyq.md) (A cable that enables CCD (Closed Case Debug))
+* Take out a WP screw
+* [Unplug your battery](battery.md)
+* Bridge two jumpers
+* [Use a SuzyQable](suzyq.md) (A cable that enables CCD (Closed Case Debugging))
```

Comment:
Both * and - are valid markdown ul. This has no effect.

Reply:
Do you and the project have a preference?

Reply:
No preference. Tbh this is a FYI instead of a change request